### ISSUE TITLE
Using PATCH to change the name of the default FusionAuth application causes a validation error

### ISSUE THREAD

**Issue by robotdan** [FusionAuth/fusionauth-issues#1110]
### Description 
Trying to change the name of the default application in FusionAuth using `PATCH` doesn't work. 

Example `PATCH` request body:
`[PATCH] /api/application`
```
{
  "application": {
    "name" :"change it"
  }
}
```

`400` response

```
{
  "fieldErrors": {
    "application.oauthConfiguration.authorizedRedirectURLs[0]": [
      {
        "code": "[invalidURL]application.oauthConfiguration.authorizedRedirectURLs",
        "message": "Invalid URL [/login] for property [application.oauthConfiguration.authorizedRedirectURLs]."
      }
    ],
    "application.oauthConfiguration.logoutURL": [
      {
        "code": "[invalidURL]application.oauthConfiguration.logoutURL",
        "message": "Invalid URL [/] for property [application.oauthConfiguration.logoutURL]."
      }
    ]
  }
}
```

### Expected behavior
Just the name change please.

**Comment by robotdan** [comment]
https://github.com/FusionAuth/fusionauth-api/pull/11
https://github.com/FusionAuth/fusionauth-app/pull/33

**Comment by mgetka** [comment]
This also resolves #534.

**Comment by robotdan** [comment]
Ah, yes - thank you @mgetka was thinking #534 was related to the merge handling with `PATCH`. Nice, I'll close that one out.
